src/scripts/syncBreaches.js

```diff
@@ -115,9 +121,12 @@ if (seen.size !== breaches.length) {
   // get
   const result = await getAllBreaches()
   console.log(result.length)
-  process.exit()
 }
 
+await getBreachIcons(breaches)
+process.exit()
+
+
```

Comment:
Since there's no code following this, does this even do anything? (If so, maybe warrants a comment on why this is there?)

```suggestion
```

Reply:
Without the `process.exit()`, the script will actually hang